browser_patches/tools/check_cdn.sh

```diff
@@ -24,36 +24,34 @@ HOST="https://playwright2.blob.core.windows.net/builds"
 FFOX_REVISION=$(head -1 ../firefox/BUILD_NUMBER)
 FFOX_ARCHIVES=(
   "$HOST/firefox/%s/firefox-mac.zip"
-  "$HOST/firefox/%s/firefox-linux.zip"
+  "$HOST/firefox/%s/firefox-ubuntu-18.04.zip"
   "$HOST/firefox/%s/firefox-win32.zip"
   "$HOST/firefox/%s/firefox-win64.zip"
 )
 FFOX_ALIASES=(
   "FF-MAC"
-  "FF-LINUX"
+  "FF-UBUNTU-18.04"
   "FF-WIN32"
   "FF-WIN64"
 )
 
 WK_REVISION=$(head -1 ../webkit/BUILD_NUMBER)
 WK_ARCHIVES=(
-  "$HOST/webkit/%s/minibrowser-gtk.zip"
-  "$HOST/webkit/%s/minibrowser-wpe.zip"
-  "$HOST/webkit/%s/minibrowser-gtk-wpe.zip"
+  "$HOST/webkit/%s/minibrowser-gtk-wpe-ubuntu-18.04.zip"
```

Comment:
Can it be just minibrowser-ubuntu-* ?

Reply:
i'll normalize all names in a follow-up.